src/buckets/components/createBucketForm/BucketOverlayForm.tsx

```diff
@@ -181,6 +181,10 @@ export default class BucketOverlayForm extends PureComponent<Props> {
       return 'This bucket needs a name'
     }
 
+    if (name.trim().length === 0) {
+      return 'Bucket name cannot be only whitespaces'
```

Comment:
@daviddkkim what do you think of this message? I think we might tweak it for the average user to something like:

>The bucket name can't be empty
or
>Please enter a name with at least $minimum_characters_allowed letters

Reply:
That is a good point! I didn't think of that..

Reply:
I like `Please enter a name with at least $minimum_characters_allowed letters` (or numbers? assuming bucket names can just be numbers)  because its action oriented.

Reply:
I can do that!

Reply:
What is the minimum number of characters we want the bucket name to be?

Reply:
I'm not sure- I think whatever the API allows should be good here